monitors/index.js

```diff
@@ -256,11 +262,11 @@ const Poll = async function(callback) {
       uniswapPriceFeedConfig,
       medianizerPriceFeedConfig
     );
-  } catch (err) {
```

Comment:
This was an evil typo

Reply:
jeez! great catch with this. no wonder we were getting such inconsistent reporting in so many cases form this bot.